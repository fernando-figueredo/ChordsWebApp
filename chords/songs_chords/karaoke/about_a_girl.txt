c c d d am am bm d c am bm bm am am d bm c am bm bm am bm bm bm am bm bm bm am f 