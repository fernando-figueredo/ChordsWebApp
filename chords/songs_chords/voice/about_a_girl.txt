bm c c bm bm c c bm em bm c bm am c c bm bm c c bm f c c bm bm c e bm e c c bm bm c c 