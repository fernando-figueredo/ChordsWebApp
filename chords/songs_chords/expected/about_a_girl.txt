em em g g em em g g em em g g em em g g em em g g em em g g em em 